pkgs/by-name/de/decktape/package.nix

```
@@ -0,0 +1,44 @@
+{
+  fetchFromGitHub,
+  buildNpmPackage,
+  lib,
+  ungoogled-chromium,
+}:
+buildNpmPackage rec {
+  name = "decktape";
+  version = "3.14.0";
+
+  src = fetchFromGitHub {
+    owner = "astefanutti";
+    repo = "decktape";
+    rev = "v{version}";
```

```suggestion
    rev = "v${version}";
```
Seems like the package was failing to build locally as well?

---

It actually builds fine and works with that `rev` value, but good catch. Fixed.